contents/docs/product-analytics/toolbar.mdx

```diff
@@ -8,38 +8,147 @@ availability:
     enterprise: full
 ---
 
-In simple terms, the Toolbar is like 'Inspect Element' in a dev tool, but for user behavior.
-
-When enabled, the Toolbar appears as an overlay **on your product or website**. It is not visible to your users. With it you can seamlessly interact with the elements to [create Actions](/tutorials/how-to-capture-events-the-easy-ways), get information about the frequency of events, visualize statistics via a heatmap, and more.
+The Toolbar is like the 'Inspect Element' dev tool, but specifically for PostHog features and data. When enabled, the Toolbar appears as an overlay **on your product or website** (it's not visible to your users). With it you can seamlessly interact with the elements to [create Actions](/tutorials/how-to-capture-events-the-easy-ways), visualize heatmaps, override feature flags, and more.
 
 ![PostHog toolbar](../../images/docs/product-analytics/toolbar.png)
 
-> For an in-depth tutorial on how to visualize user behavior, check out our [Toolbar tutorial](/docs/tutorials/toolbar)
-
-## How to activate the Toolbar
+## How to launch the Toolbar
 
 1. Go to 'Launch Toolbar' in the left-hand menu
 2. Click 'Enable PostHog Toolbar'
-3. Add the URLs you wish to enable the toolbar for. Only you will be able to see it (and not your users).
+3. Add the URLs you wish to enable the toolbar for. Only you will be able to see it – not your users.
 4. Click Save and then Launch.    
 5. A new window will open with toolbar on your website. Click on the toolbar to interact with it.
 
 ![How to setup the PostHog toolbar](../../images/docs/product-analytics/toolbar-setup.mp4)
```

Comment:
This isn't loading for me for some reason.

Reply:
Not sure why, but I checked now and its showing in the Vercel preview - https://posthog-git-update-heatmap-and-toolbar-docs-post-hog.vercel.app/docs/product-analytics/toolbar